Source/WebKit/WebProcess/Network/webrtc/LibWebRTCSocket.cpp

```diff
@@ -142,18 +142,16 @@ int LibWebRTCSocket::Close()
 
 int LibWebRTCSocket::GetOption(rtc::Socket::Option option, int* value)
 {
-    ASSERT(option < MAX_SOCKET_OPTION);
-    if (auto storedValue = m_options[option]) {
-        *value = *storedValue;
-        return 0;
-    }
-    return -1;
+    auto iterator = m_options.find(option);
+    if (iterator == m_options.end())
+        return false;
+
+    *value = iterator->second;
+    return true;
```

Comment:
This used to return 0 when option is found, now it returns non-zero (1); seems reversed

Reply:
Is there a reason to not return 0 and -1 as before?

Reply:
Indeed, I changed it back though I do not see where this return value is used (and libwebrtc is not totally consistent in GetOption either).